content/en/docs/devops-user-guide/how-to-use/jenkins-setting.md

```diff
@@ -8,35 +8,43 @@ Weight: 11240
 
 Jenkins is powerful and flexible and it has become the de facto standard for CI/CD workflows. Nevertheless, many plugins require users to set system-level configurations before they can be put to use.
 
-The KubeSphere DevOps System offers containerized CI/CD functions based on Jenkins. To provide users with a schedulable Jenkins environment, KubeSphere uses **Configuration-as-Code** for Jenkins system settings, which requires users to log in to the Jenkins dashboard and reload the configuration after it is modified. In the current release, Jenkins system settings are not available on the KubeSphere console, which will be supported in upcoming releases.
+The KubeSphere DevOps System offers containerized CI/CD functions based on Jenkins. To provide users with a schedulable Jenkins environment, KubeSphere uses **Configuration as Code** for Jenkins system settings, which requires users to log in to the Jenkins dashboard and reload the configuration after it is modified. In the current release, Jenkins system settings are not available on the KubeSphere console, which will be supported in upcoming releases.
 
 This tutorial demonstrates how to set up Jenkins and reload configurations on the Jenkins dashboard.
 
 ## Prerequisites
 
 You have enabled [the KubeSphere DevOps System](../../../pluggable-components/devops/).
 
+## Jenkins Configuration as Code
+
+KubeSphere has Jenkins Configuration as Code plugin installed by default to allow you to define the desired state of your Jenkins configuration through YAML files and makes it easy to reproduce your Jenkins configurations including plugin configurations. You can find descriptions about specific Jenkins configurations and example YAML files [in this directory](https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos).
+
+Besides, you can find the `formula.yaml` file in the repository [ks-jenkins](https://github.com/kubesphere/ks-jenkins), where you can view plugin versions and customize these versions based on your needs.
+
+![plugin-version](/images/docs/devops-user-guide/using-devops/jenkins-system-settings/plugin-version.png)
+
 ## Modify the ConfigMap
 
-It is recommended that you configure Jenkins in KubeSphere through Configuration-as-Code (CasC). The built-in Jenkins CasC file is stored as a [ConfigMap](../../../project-user-guide/configuration/configmaps/).
+It is recommended that you configure Jenkins in KubeSphere through Configuration as Code (CasC). The built-in Jenkins CasC file is stored as a [ConfigMap](../../../project-user-guide/configuration/configmaps/).
 
 1. Log in to KubeSphere as `admin`. Click **Platform** in the top left corner and select **Cluster Management**.
 
-   ![cluster-management](/images/docs/devops-user-guide/using-devops/jenkins-system-settings/cluster-management.jpg)
+   ![cluster-management](/images/docs/devops-user-guide/using-devops/jenkins-system-settings/cluster-management.png)
```

Comment:
This image is not necessary. Remove it. Note that you only need to put images for key steps in a document.